src/grt/src/fastroute/src/utility.cpp

```diff
@@ -964,7 +964,7 @@ void FastRouteCore::layerAssignmentV4()
         treenodes[n2a].assigned = true;
 
       }  // edge len > 0
-    }    // eunmerating edges
+    }  // eunmerating edges
```

Comment:
These whitespace changes seem unrelated.

Reply:
this is probably form the clang-format version issue I mentioned. I'll need to redo the formatting on an older version than what is currently installed.